io/src/error.rs

```diff
@@ -6,10 +6,12 @@ use core::fmt;
 #[derive(Debug)]
 pub struct Error {
     kind: ErrorKind,
-    /// Indicates that the `struct` can pretend to own a mutable static reference
-    /// and an [`UnsafeCell`](core::cell::UnsafeCell), which are not unwind safe.
-    /// This is so that it does not introduce non-additive cargo features.
-    _not_unwind_safe: core::marker::PhantomData<(&'static mut (), core::cell::UnsafeCell<()>)>,
+    /// We want this type to be `?UnwindSafe` and `?RefUnwindSafe` - the same as `std::io::Error`.
```

Comment:
`!`, not `?`

Reply:
Gonna go ahead and merge this -- we can fix this private doccomment in a followup.

Reply:
Kix!! Funny typo, I guess I mixed this up with `?Sized`